backend/fms_core/models/sample.py

```diff
@@ -159,35 +54,9 @@ class Meta:
     def is_depleted(self) -> str:
         return "yes" if self.depleted else "no"
 
-    # Computed properties for individuals
-
-    @property
-    def individual_name(self) -> str:
-        return self.individual.name if self.individual else ""
-
-    @property
-    def individual_sex(self) -> str:
-        return self.individual.sex if self.individual else ""
-
-    @property
-    def individual_taxon(self) -> str:
-        return self.individual.taxon if self.individual else ""
-
-    @property
-    def individual_cohort(self) -> str:
-        return self.individual.cohort if self.individual else ""
-
-    @property
-    def individual_pedigree(self) -> str:
-        return self.individual.pedigree if self.individual else ""
-
     @property
-    def individual_mother(self) -> Optional["Individual"]:
-        return self.individual.mother if self.individual else None
-
-    @property
-    def individual_father(self) -> Optional["Individual"]:
-        return self.individual.father if self.individual else None
+    def is_pool(self) -> bool:
+        return self.derived_samples.objects.count() > 1
```

Comment:
Would the self.derived_samples association work directly?
You could simply check if there are many DerivedBySample with this self (sample).id

Reply:
This is less simple but maybe a bit more performant ... i'll change it and comment...